app/controllers/estimates_controller.rb

```diff
@@ -1,22 +1,17 @@
 class EstimatesController < ApplicationController
   def new
-    redirect_to estimate_build_estimates_path cfe_connection.create_assessment_id
+    redirect_to estimate_build_estimates_path SecureRandom.uuid
   end
 
   def create
-    # The last thing to do before retrieving the result is to tell CFE
-    # about the applicant
-    create_applicant
-
-    @model = cfe_connection.api_result(params[:cfe_id])
```

Comment:
Since we're not doing this, we can remove `def cfe_connection` from ApplicationController, which is one of the places where we don't have line coverage